src/components/ServiceDetails/AboutService/ServiceDemo/ThirdPartyServiceErrorBoundary.js

```diff
@@ -0,0 +1,33 @@
+import React, { Component } from "react";
+
+class ThirdPartyServiceErrorBoundary extends Component {
+  state = {
+    error: undefined,
+    info: "",
+  };
+
+  static getDerivedStateFromError = error => {
+    return { error };
+  };
+
+  componentDidCatch = (error, info) => {
+    this.setState({ error, info });
+  };
+
+  render() {
```

Comment:
Shall we get the design team to give us a nice UI for this?

Reply:
Sure.. will raise this concern with deesign team